docs/community.md

```diff
@@ -1,23 +1,44 @@
 # Community Policy
 
-We craft Ludwig with love and care, at the best of our skills and knowledge, but this project would not be possible without the contribution of an incredible community.
-With the support of the community we were able to improve the quality of Ludwig dramatically, as members provided both fixes, new features and helped directing the developments.
-We are really grateful for that and in exchange we strive to make the development process as open as possible and communication with the development team easy and direct.
-We strive to create an inclusive community where everyone is welcome as long as they respect everyone else, so harassment and any other form of non inclusive behavior will not be allowed.
+We craft Ludwig with love and care, to the best of our skills and knowledge, and
+this project would not be possible without the contribution of an incredible
+community.
+
+Members of the Ludwig community provide fixes, new features, functionality, and
+documentation, which not only improves Ludwig but also shapes technical
+direction.
+
+We are really grateful for that and in exchange we strive to make the
+development process as open as possible and communication with the development
+team easy and direct.
+
+We strive to create an inclusive community where everyone is respected.
+Harassment and any other form of non-inclusive behavior will not be tolerated.
 
 # Issues
 
-If you encounter an issue when using Ludwig, please add it to our [GitHub Issues tracker](https://github.com/ludwig-ai/ludwig/issues).
-Please make sure we are able to replicate the issue by providing wither model definition + command + data or code + data.
-If your data cannot be shared, please use the `dataset_synthesizer` script available in `ludwig/data/dataset_synthesizer` to create a synthetic data with the same feature types.
+If you encounter an issue when using Ludwig, please add it to our
+[GitHub Issues tracker](https://github.com/ludwig-ai/ludwig/issues).
+Please make sure we are able to replicate the issue by providing the model
+definition + command + data or code + data.
+
+If your data cannot be shared, please use the `dataset_synthesizer` script
+available in `ludwig/data/dataset_synthesizer` to create a synthetic data with
+the same feature types.
```

Comment:
We since added a command for it and documentation, we should replace with the command and point to the docs ;)

Reply:
Done.